sql-composer/src/parser.rs

```diff
@@ -195,12 +202,12 @@ named!(
                 p.expect("unable to build ParsedItem of String from column_list parser")
             })
           ),
-          opt!(do_parse!(multispace0 >> tag!(",") >> multispace0 >> ()))
+          opt!(comma_padded)
     )
 );
 
 named!(take_while_name_char(Span) -> Span,
-    do_parse!(
+    complete!(do_parse!(
```

Comment:
I don't think this `complete!` will help because of how `take_while1!` macro is the streaming version.

Though I'd be happy to be wrong on that :)

Reply:
I don't recall if this actually ended up necessary, I think it just ended up something I tinkered with and should not have committed though.